wgpu-core/src/command/bundle.rs

```diff
@@ -678,9 +681,9 @@ where
 
 /// Error encountered when finishing recording a render bundle.
 #[derive(Clone, Debug, Error)]
-#[error("{command}")]
+#[error("{ctx}")]
 pub struct RenderBundleError {
-    command: &'static str,
+    ctx: PassErrorCtx,
```

Comment:
maybe we should call it `scope` with `ErrorScope` instead?

Reply:
Sure!